packages/vuepress-theme-odyssey/styles/_Visual.scss

```diff
@@ -0,0 +1,82 @@
+.docskit-visual {
+  grid-column: 2 / 3;
+  min-width: 420px;
```

Comment:
Is this nec? The parent grid has a min-width here of the $max-line-length, so we shouldn't need to set this.

Reply:
420 is admittedly a weird, and relatively arbitrary value. Without min-width here this happens: 

![image](https://user-images.githubusercontent.com/62259644/98616573-edda6200-22b1-11eb-88c2-9517220fb5dc.png)

Reply:
I think we have an issue at the parent grid level. Okay to ship with this, but will you create a ticket to revisit?